appveyor-e2e.bat

```diff
@@ -2,11 +2,11 @@
 IF not defined APPVEYOR_PULL_REQUEST_NUMBER (
   IF not defined BUILDKITE_PULL_REQUEST (
     REM Run full test suite if not on a Appveyor/Buildkite PR.
-    node tests\legacy-cli\run_e2e.js --appveyor "--glob=tests/{basic,commands,generate,build/styles}/**"
+    node tests\legacy-cli\run_e2e.js --appveyor "--glob=tests/{basic,commands,generate,build/styles/ivy}/**"
```

Comment:
Shouldn't this be `,ivy` instead of `/ivy`?

Reply:
Yes, totally fixed.